qa/standalone/misc/test_failure.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
I removed that script from standalone/misc.  I'll but it somewhere that won't be run by teuthology.  It isn't part of this pull request anymore.